### PR TITLE
added newest pre-releases to changelog

### PR DESCRIPTION
**Pull request by InQuognito**
pre2, pre3, and pre4 all added

### CODE REVIEW

File: 1.20.5/1.20.5-pre2.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+advancement | `Snip it!` advancement is renamed to `Shear Brilliance`
+
+data | Using ctrl+pick block no longer lists "(+NBT)" in the item tooltip
```
Comment (misode) [inline 1.20.5/1.20.5-pre2.md:3]: This should have categories `item` and `nbt`. `data` is for changes to the builtin vanilla data pack.

File: 1.20.5/1.20.5-pre2.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+assets | The animated Nether Portal texture is displayed when changing dimension to or from The Nether
```
Comment (misode) [inline 1.20.5/1.20.5-pre2.md:5]: Everything should be lowercase (also in other places)

File: 1.20.5/1.20.5-pre2.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+assets | The animated End Portal effect is displayed when changing dimension to or from The End
+
+pack breaking | The Data Pack version is now 40
```
Comment (misode) [inline 1.20.5/1.20.5-pre2.md:9]: This should always be the first entry in the file. Also look at some of the other files to match the sentence.

File: 1.20.5/1.20.5-pre3.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+pack breaking | The Data Pack version is now 41
+
+tag | Replaced Entity Type tag `minecraft:punchable_projectiles` with tag `minecraft:redirectable_projectile` for projectiles which can be redirected by player attacks and projectiles
```
Comment (misode) [inline 1.20.5/1.20.5-pre3.md:3]: Can have categories `data` and `tag`. Also don't include `minecraft:`

File: 1.20.5/1.20.5-pre4.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+pack breaking | The Resource Pack version is now 32
+
+font | The `shift` in TTF glyph providers is now restricted to the range `[-512; 512]`
```
Comment (misode) [inline 1.20.5/1.20.5-pre4.md:3]: Should have tags `assets`, `font`, and `breaking`.

**Review by misode — CHANGES_REQUESTED** [review]
Thanks for contributing! I made some comments to make them a bit more consistent.

File: 1.20.5/1.20.5-pre4.md (hunk 5 above)
Comment (InQuognito) [reply 1.20.5/1.20.5-pre4.md:3]: fixed

File: 1.20.5/1.20.5-pre3.md (hunk 4 above)
Comment (InQuognito) [reply 1.20.5/1.20.5-pre3.md:3]: fixed.

File: 1.20.5/1.20.5-pre2.md (hunk 1 above)
Comment (InQuognito) [reply 1.20.5/1.20.5-pre2.md:3]: fixed. thank you for all the feedback!

File: 1.20.5/1.20.5-pre2.md (hunk 2 above)
Comment (InQuognito) [reply 1.20.5/1.20.5-pre2.md:5]: fixed.

File: 1.20.5/1.20.5-pre2.md (hunk 3 above)
Comment (InQuognito) [reply 1.20.5/1.20.5-pre2.md:9]: fixed.

**Comment by InQuognito** [comment]
changes implemented, as well as adding rc1. Thanks for the feedback!

**Comment by misode** [comment]
Thanks a lot! ❤️

**Pull request merged**